apps/pm-user/src/components/organisms/sideBar/SideBar.js

```diff
@@ -1,28 +1,35 @@
 import React from 'react';
+import { DEFAULT_SELECTED_ITEM_KEY, MENU_ITEMS } from './constants';
 import { Menu } from '../../atoms';
-import { ProfileSummary } from "../../molecules";
+import { ProfileSummary } from '../../molecules';
+import { QuickInfoBar } from '../../molecules';
 import styles from './sideBar.module.scss';
-import { DEFAULT_SELECTED_ITEM_KEY, MENU_ITEMS } from './constants';
 
 function SideBar() {
-
   const handleMenuItemClick = ({ key }) => {
     //some code here
   };
```

Comment:
Is this function even required ?

Reply:
this is created by Chitanaya not me

Reply:
Ok... understood... Chaitanya wanted us to figure out how to redirect...
Use my code below... and update the function...
Then links will work...
Right ow all links are not working....
![image](https://user-images.githubusercontent.com/48716076/185747427-fd945c4e-fdbd-4369-a6d4-f816ae690384.png)